setup.cfg

```diff
@@ -105,4 +105,7 @@ ignore_missing_imports = True
 ignore_errors = True
 
 [mypy-psycopg2]
-ignore_errors = True
+ignore_missing_imports = True
```

Comment:
I think we can. Weather station used to use this db, but I am not sure about the carpark agent.